### ISSUE TITLE
How to deal with video with height=768 and width=432

### ISSUE THREAD

**Issue by CrossLee1** [ykasten/layered-neural-atlases#4]
Thanks for your excellent work and source code.

I saw that in your provided `config.json`, height is 432 and width is 768. 
But in my case, the height is 768 and width is 432, and I only changed the corresponding config file as:
`"resx": 768,
"resy": 432,`
However, the reconstructed video seem strange and cannot work well.

Wonder if there are other codes I should change when I deal with this portrait video.
Thanks a lot ~

**Comment by dolev104** [comment]
For a portrait video as you described you would need to define "resx": 432, "resy": 768

**Comment by apletea** [comment]
On a portrait video there is a bug that it do wrong mapping

**Comment by eyaler** [comment]
@apletea did you ever solve this?

**Comment by apletea** [comment]
> @apletea did you ever solve this?

Nope.Decided to rescale all vertical video to horizontal. Anyway steel it's not an only problem of aproach. Later faced that mask mapper didn't learns properly and it's much better to cut of model for mask mapping and use some 3rd solution to predict mask for objects
